bumble/hfp.py

```diff
@@ -20,7 +20,8 @@
 import collections
 
 from .colors import color
-
+from .rfcomm import DLC
+from typing import Union
```

Comment:
style: we usually try to keep the imports grouped, with the builtin modules first, followed by the 3rd party modules, then finally the local ones. So I'd recommend moving the typing import up to line 21

Reply:
Done